app/controllers/PromoLandingPage.scala

```diff
@@ -81,14 +80,14 @@ object PromoLandingPage extends Controller {
     } else if ((applicableRatePlanIds intersect paperOnlyPackageRatePlanIds).nonEmpty) {
       Some(routes.Shipping.viewCollectionPaper())
     } else if ((applicableRatePlanIds intersect guardianWeeklyRatePlanIds).nonEmpty) {
-      Some(routes.WeeklyLandingPage.index(maybeCountry))
+      Some(routes.WeeklyLandingPage.index(Some(country)))
```

Comment:
If we're not going to delegate country lookup to WeeklyLandingPage then you can use WeeklyLandingPage.withCountry instead.